tests/system/provisioning/basic_cluster/playbook.yml

```diff
@@ -136,3 +124,15 @@
     - ./vars/configurations.yml
   roles:
     - name: "roles/agent-role"
+
+- name: Wazuh Agent2
+  hosts: wazuh-agent2
+  vars:
+    manager_hostname: wazuh-worker1
+    agent_id: "{{ agent2_id }}"
+    agent_hostname: "{{ agent2_hostname }}"
+    agent_key: "{{ agent2_key }}"
+  vars_files:
+    - ./vars/configurations.yml
+  roles:
+    - name: "roles/agent-role"
```

Comment:
This was used before as a mechanism to avoid having the third agent in `never_connected` status a few seconds after the environment was up. However, as the code was modified to check the agents' status before running the test this is not necessary.